projects/packages/sync/src/replicastore/class-table-checksum.php

```diff
@@ -304,6 +304,30 @@ protected function get_default_tables() {
 				'table_join_field'          => 'order_item_id',
 				'is_table_enabled_callback' => array( $this, 'enable_woocommerce_tables' ),
 			),
+			'wc_orders'                  => array(
+				'table'                     => "{$wpdb->prefix}wc_orders",
+				'range_field'               => 'id',
+				'key_fields'                => array( 'id', 'parent_order_id' ),
```

Comment:
This should be fine to be just `id`. The orders should be unique by `id` if I recall correctly. No need to make a compound key if the entries are going to be 1:1 between the Jetpack site and WPCOM. Especially for non-meta tables.